.github/workflows/ci.yaml

```
@@ -219,9 +219,9 @@ jobs:
       - name: Build subtree
         run: yarn workspaces foreach -Rptv --jobs unlimited --from ${{ matrix.package }} run build
       - name: Set up tests (if script exists)
-        run: yarn workspaces foreach --include ${{ matrix.package }} run setup-test
+        run: yarn workspaces foreach -W --include ${{ matrix.package }} run setup-test
       - name: Run tests (if script exists)
-        run: yarn workspaces foreach --include ${{ matrix.package }} run test
+        run: yarn workspaces foreach -W --include ${{ matrix.package }} run test
```

Alternatively:

```suggestion
        run: yarn workspaces foreach --worktree --include ${{ matrix.package }} run setup-test
- name: Run tests (if script exists)
        run: yarn workspaces foreach --worktree --include ${{ matrix.package }} run test
```

This used to be implicit, but now one of `--all`, `--recursive` or `--worktree` is required